tests/cli/base.py

```diff
@@ -74,3 +81,160 @@ def command_test(self, arg, kwarg=None, kwarg2=False):
                 pass
         argv = 'mage.py test'
         self.assertRaises(SystemExit, lambda: manage(dict(test=TestCommand()), argv.split()))
+
+    def test_digest_not_found(self):
+        class TestCommand(Cli):
+            def command_test(self, kwarg=None, kwarg2=False):
+                pass
+        argv = 'mage.py test1:test --kwarg --kwarg2'
+
+        out = StringIO()
+        with patch.object(sys, 'stdout', out):
+            with self.assertRaises(SystemExit):
```

Comment:
This looks strange. If we expect command to exit, there are several ways to do so, not just calling `sys.exit()`. Do we expect some particular or just non-zero exit code? Then we should check it too. Probably, `sys.exit()` is the only good way to exit with non-zero  code, so expecting exception is ok in this case.